projects/linux/Makefile

```diff
@@ -24,10 +24,7 @@ INCLUDES=					\
 
 EXTRA_CFLAGS +=-Wall
 EXTRA_CFLAGS +=-Werror
-EXTRA_CFLAGS +=-Wno-switch-break
-EXTRA_CFLAGS +=-Wno-pointer-to-int-cast
```

Comment:
The "-Wno-pointer-to-int-cast" flag is required for one of our commercial tests, so we will maintain this flag. Removing "-Wno-switch-break" and "-Wno-error=unused-variable" seems to be a good decision.